lib/uwb/UwbDevice.cxx

```diff
@@ -61,7 +61,8 @@ UwbDevice::OnSessionStatusChanged(UwbSessionStatus statusSession)
         return;
     }
 
-    // TODO: implement this
+    PLOG_VERBOSE << "session changed state:" << session->GetSessionStatus().ToString() << " --> " << statusSession.ToString();
```

Comment:
This log would be better suited in `UwbSession::SetSessionStatus()` since then it will cover when the status is changed from _every_ source (eg. including unit test sources), instead of just from `UwbDevice`.